_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -112,7 +113,35 @@ class HealthCareApp extends React.Component {
           },
           childInformation: {},
           annualIncome: {},
-          deductibleExpenses: {}
+          deductibleExpenses: {},
+        },
+
+        // TODO: insuranceInformation should be an array where each row
+        //       contains all the following fields:
+        insuranceInformation: {
+          insuranceInfo: {
+            isCoveredByHealthInsurance: false,
+            insuranceName: undefined,
```

Comment:
Sorry, that's on me, I told Joe to initialize these as undefined. I've been doing that and haven't had a problem, but I might be missing something?